system-test/integration_test.sh

```diff
@@ -23,13 +23,9 @@ export GCLOUD_TESTS_NODEJS_PROJECT_ID="cloud-profiler-e2e"
 export GCLOUD_TESTS_NODEJS_ZONE="us-east4-b"
 export GOOGLE_APPLICATION_CREDENTIALS="${SERVICE_KEY}"
 
-# Move test to go path.
-export GOPATH="$HOME/go"
```

Comment:
GOPATH is not needed when using modules? Maybe we should have stopped using it in the probers too then?

Reply:
Done.